src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -472,12 +472,22 @@ protected virtual void SweepMethods (Collection<MethodDefinition> methods)
 		void SweepOverrides (MethodDefinition method)
 		{
 			for (int i = 0; i < method.Overrides.Count;) {
-				if (Context.Resolve (method.Overrides[i]) is MethodDefinition ov && ShouldRemove (ov))
+				if (Context.Resolve (method.Overrides[i]) is MethodDefinition ov && ShouldRemove (ov) && !IgnoreScope (ov.DeclaringType.Scope))
```

Comment:
Can we avoid checking each overridden method individually by sweeping the overrides themselves, instead of sweeping the overrides from the base type?